application/routes/vnc.py

```
@@ -36,16 +37,19 @@ def start_vnc():
     session_id = request.json.get('sessionID')
     load_check = request.json.get('loadCheck', True)
     load_credentials = request.json.get('loadCredentials', True)
-
+    logger.debug(f"Starting VNC session: {session_id} with load check: {load_check} and load credentials: {load_credentials}")
```

Ruff

Logging statement uses f-string

(G004)